src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
The class Circle should not extend Figure directly if Figure is an abstract class meant to define behavior. Instead, consider using an interface to define the behavior that Circle should implement if the behavior is shared across multiple classes.